spec/services/obituary_service_spec.rb

```diff
@@ -21,4 +21,36 @@
 
     expect(obituaries.first[:data][:attributes][:abstract].downcase.include?('coronavirus') || obituaries.first[:data][:attributes][:abstract].downcase.include?('covid')).to eq(true)
   end
+
+  it "returns recent obituaries", :vcr do
```

Comment:
Added more tests for service

Reply:
Thank you, this is awesome! Appreciate your thorough testing :grateful: